src/node.cc

```diff
@@ -3224,6 +3224,21 @@ void SetupProcessObject(Environment* env,
                     Integer::New(env->isolate(),
                                  v8::kPromiseHandlerAddedAfterReject));
 
+  // process._debugOptions
+  Local<Object> debugOptions = Object::New(env->isolate());
+  READONLY_DONT_ENUM_PROPERTY(process,
+                              "_debugOptions",
```

Comment:
Yes, please drop this off `process.binding('config')` rather than adding a new property off `process`